src/App.js

```diff
@@ -337,22 +322,16 @@ export default function App() {
             onTouchEnd={handleTouchStop}
           />
         </main>
-        <Footer
-          onClickSave={handleSave}
-          onClickLoad={handleLoad}
-          patch={savedPatch}
-          loadedPatch={loadedPatch}
-          setOsc1Frequency={setOsc1Frequency}
-          setOsc1Type={setOsc1Type}
-          setOsc2Frequency={setOsc2Frequency}
-          setOsc2Type={setOsc2Type}
-          setFilterFrequency={setFilterFrequency}
-          setFilterType={setFilterType}
-          setAmpEnvelope={setAmpEnvelope}
-          setResonance={setResonance}
-          setReverbDuration={setReverbDuration}
-          setPhaserDuration={setPhaserDuration}
-        />
```

Comment:
This was the first idea - it would have had to happen because Load/Save-functions for localStorage are executed in the Footer. The idea of higher order-functions turned out to be more practical in this case.